tests/src/FunctionalJavascript/ListPagesExposedFiltersTest.php

```diff
@@ -57,7 +62,7 @@ public function testListPagePluginFiltersFormConfiguration(): void {
     ];
     $this->assertEquals($expected_entity_types, $actual_entity_types);
     // By default, Node is selected if there are no stored values.
-    $this->assertOptionSelected('Source entity type', 'Content');
+    $this->assertSession()->optionExists('Source entity type', 'Content');
```

Comment:
It is also required to assert the attribute 'selected'.

Reply:
This will be fixed on next update.